scale/batch/test/test_views.py

```diff
@@ -860,3 +1119,63 @@ def test_create_successful(self):
+        }
+        the_dataset = data_test_utils.create_dataset(definition=dataset_def)
+        workspace = storage_test_utils.create_workspace()
+        src_file_a = storage_test_utils.create_file(file_name='input_a.PNG', file_type='SOURCE', media_type='image/png',
+                                              file_size=10, data_type_tags=['type'], file_path='the_path',
+                                              workspace=workspace)
+        src_file_b = storage_test_utils.create_file(file_name='input_b.PNG', file_type='SOURCE', media_type='image/png',
+                                              file_size=10, data_type_tags=['type'], file_path='the_path',
+                                              workspace=workspace)
+        data_list = []
+        data_dict = {
+            'version': '6',
+            'files': {'INPUT_IMAGE': [src_file_a.id]},
+            'json': {}
+        }
+        data_list.append(DataV6(data=data_dict).get_dict())
+        data_dict = {
+            'version': '6',
+            'files': {'INPUT_IMAGE': [src_file_b.id]},
+            'json': {}
+        }
+        data_list.append(DataV6(data=data_dict).get_dict())
+        member_2 = data_test_utils.create_dataset_members(dataset=the_dataset, data_list=data_list)
+        
+        recipe_type_5 = recipe_test_utils.create_recipe_type_v6()
+        recipe_type_def = definition=copy.deepcopy(recipe_test_utils.RECIPE_DEFINITION)
+        recipe_type_def['nodes']['node_d'] = {'dependencies': [{'name': 'node_c', 'acceptance': True}],
+                                                 'input': {'input_a': {'type': 'recipe', 'input': 'bar'},
+                                                           'input_b': {'type': 'dependency', 'node': 'node_c',
+                                                                       'output': 'INPUT_IMAGE'}},
+                                                 'node_type': {'node_type': 'recipe', 'recipe_type_name': recipe_type_5.name,
+                                                               'recipe_type_revision': recipe_type_5.revision_num}}
+        recipe_type_4 = recipe_test_utils.create_recipe_type_v6(definition=recipe_type_def)
+        
+        json_data = {
+            'recipe_type_id': recipe_type_4.id,
+            'definition': {
+                'dataset': the_dataset.id,
+                'forced_nodes': {
+                    'all': True,
+                },
+            },
+            'configuration': {
+                'priority': 100
+            }
+        }
+        
+        url = '/v6/batches/validation/'
+        response = self.client.generic('POST', url, json.dumps(json_data), 'application/json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK, response.content)
+        result = json.loads(response.content)
+        self.assertTrue(result['is_valid'])
+        self.assertEqual(result['recipes_estimated'], 4) # two files, each with one sub-recipe
+        
```

Comment:
If we can easily add a test somewhere that checks the created jobs' priority that would be great.  If not, that can be a separate issue.

Reply:
Added a portion to the `TestBatchesViewV6.test_create_priority` that tests the new batch with an updated priority.